packages/babel-parser/src/plugins/estree.ts

```diff
@@ -424,6 +424,9 @@ export default (superClass: typeof Parser) =>
           this.hasPlugin("importAttributes") ||
           this.hasPlugin("importAssertions")
         ) {
+          (node as N.Node as N.EstreeImportExpression).options =
+            node.arguments[1] ?? null;
+          // compatibility with previous ESTree AST
           (node as N.Node as N.EstreeImportExpression).attributes =
```

Comment:
Can we maybe avoid this in `BABEL_8_BREAKING`? (in a separate PR)

Reply:
Agreed, it was added for prettier compatibility only.